pom.xml

```diff
@@ -27,7 +27,7 @@
 		<dependency>
```

Comment:
We should increase the version of Codebase to 2.2.0, because new functionality was added.